util.grace

```diff
@@ -396,31 +396,38 @@ method execDir {
     execDirCache
 }
 
-method file(name) on(origin) orPath(pathString) otherwise(action) {
+method files(names) on(origin) orPath(pathString) otherwise(action) {
     def locations = filePath.split(pathString)
```

Comment:
I'm guessing that this method is intended to do something different from repeated requests on `file(_)on(_)orPath(_)`.  Is there an implicit intention that all of the found files should be in the same directory?  Or what?  

Please add a purpose statement immediately after the method header explains what this method does.  And perhaps change the name of the method — the name now says that it returns `files`, but it seems not to.